### DISCUSSION TITLE
Possible to use this method to instal Vitis?

### DISCUSSION THREAD

**Discussion by JJ1553** [ichi4096/vivado-on-silicon-mac#40] [Q&A]
Im in an fpga class for college and im at the point where I really need vitis installed, no version of a VM works. Is there any way to adapt these installs for vivado to vitis?

**Comment by ichi4096** [comment]
There may be a way but these scripts just provide a Vivado installation. What you can do is install Vitis in the GUI after installing Vivado. Maybe I'll add an option to install Vitis later.
